packages/foreman/foreman/dynflow@.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Foreman jobs daemon on sidekiq
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
+User=foreman
+TimeoutSec=90
+ExecStart=/usr/bin/bundle exec sidekiq -e ${RAILS_ENV} -r ${DYNFLOW_SIDEKIQ_SCRIPT} -C /etc/foreman/dynflow/%i.yml
+WorkingDirectory=/usr/share/foreman
+EnvironmentFile=-/etc/sysconfig/dynflow
```

Comment:
For new services we don't use `/etc/sysconfig` anymore but prefer systemd drop in configs. This saves us from having to manage the difference between RPMs (/etc/sysconfig) and debs (/etc/default).